package.json

```diff
@@ -17,6 +17,8 @@
     "@mui/material": "^5.14.7",
     "@types/react-router": "^5.1.20",
     "@types/react-router-dom": "^5.3.3",
+    "envalid": "^7.3.1",
+    "keycloak-js": "^22.0.1",
```

Comment:
´keycloak-js` versioning follows the versioning of Keycloak itself. e.g. this version is meant for Keycloak v. 22.0.1. Typically they've been backwards compatible but just in case I'd use the correct version. Timebank staging Keycloak seems to be version 19.0.2.
The Keycloaks version should be bumped to the most up to date (22.0.1).

Reply:
Fixed